packages/module/patternfly-docs/content/extensions/chatbot/examples/UI/UI.md

```diff
@@ -371,6 +371,16 @@ If you're showing a conversation that is already active, you can set the `active
 
 ```
 
+### Resizable drawer
+
+By default, the drawer is fixed width. We apply a focus trap and a default size of 384px to the drawer. The drawer can also accept additional props from PatternFly to make it resizable, or even other sizes. Making the drawer resizable can be helpful for embedded ChatBots on tablet-sized devices or smaller browser windows, where screen real estate is at a premium. (By default, PatternFly drawers are open to their full width on mobile.)
+
+In this example, the drawer can be resized up the max size of the parent and down to 200px wide. Further customization is possible. For more information on what props you can pass, see the [PatternFly Drawer DrawerPanelContent props](/components/drawer/#drawerpanelcontent).
```

Comment:
```suggestion
By default, the conversation history drawer has a fixed width (384px) and a focus trap. To provide users with more flexibility as they navigate their conversation history, or to better support embedded ChatBots on tablet-sized devices or smaller browser windows, you can instead make the drawer resizable. By default, even resizable drawers will still open to their full width on mobile devices.

In this example, the drawer can be resized up to the max size of the parent and resized down to 200px wide. To customize this behavior further, including width, style, focus behavior, and more, use [PatternFly `<DrawerPanelContent>` props](/components/drawer/#drawerpanelcontent).
```
lmk what you think about how I phrased this, and please call out if I misspoke!

Reply:
Also, with the last line, I can see that we're importing a handful of those extra drawer components. Should we mention more of them here intentionally?

Reply:
I merged what you had there just now. I mean, we could? They're just buckets of props defined in PatternFly. I basically spread props down for everything we pull from PatternFly that's part of the drawer. (The drawer has a lot of subcomponents.)

Maybe something like this? What do you think?
To customize this behavior further (including width, style, and focus behavior, for example) use [PatternFly `<Drawer>` and `<DrawerPanelContent>` props](/components/drawer). You can also pass props down to other subcomponents of the drawer using props like `DrawerPanelContentProps`.